violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/behavior/ShowMenuOnRightClickBehavior.java

```diff
@@ -72,6 +72,20 @@ private JPopupMenu getPopupMenu() {
      */
     private JPopupMenu fillMenu(JPopupMenu aPopupMenu)
     {
+    	select.addActionListener(new ActionListener(){
+    		public void actionPerformed(ActionEvent event)
+    		{
+    			IEditorPartBehaviorManager behaviorManager = ShowMenuOnRightClickBehavior.this.editorPart.getBehaviorManager();
+    			List<SelectSelectToolBehavior> found = behaviorManager.getBehaviors(SelectSelectToolBehavior.class);
+    			
+                if (found.size() != 1) {
+                    return;
```

Comment:
We try to fix those wherever they're encountered for readability sake.
On the side note : Is there any sane reason to force return in void method I fail to see? or is it just bad craftsmanship? [Citation needed]